include/api/netinet6/__in6.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2013 Cloudius Systems, Ltd.
```

Comment:
Umm...  wrong copyright, I think...

Reply:
File was originally copied from include/api/netinet/__in.h and includes that copyright.
Should I take it out completely or just update to 2018?